agimus_demos_common/launch/franka_common.launch.py

```diff
@@ -122,13 +123,37 @@ def launch_setup(
         value_type=str,
     )
 
+    srdf_file_subtitution = PathJoinSubstitution(
+        [
+            FindPackageShare("franka_description"),
+            "robots",
+            arm_id_str,
+            f"{arm_id_str}.srdf",
+        ]
+    )
+    srdf_file = srdf_file_subtitution.perform(context)
+    with open(srdf_file, "r") as f:
+        robot_srdf_description = f.read()
+
     robot_state_publisher_node = Node(
         package="robot_state_publisher",
         executable="robot_state_publisher",
         parameters=[get_use_sim_time(), {"robot_description": robot_description}],
         output="screen",
     )
 
+    robot_srdf_description_publisher = ExecuteProcess(
```

Comment:
Can you check the QoS for this publisher?
This is very important for ROS2
And that you are not streaming this data but sending it on demand.

Reply:
I'll remove the SRDF stuff from this PR. Theo will add it in its environment PR.